package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/mapbox-events-android",
-  "version": "4.0.0",
+  "version": "4.7.1",
```

Comment:
4.7.0?

Reply:
4.7.0 release is already done. Whatever merged to master from now on will be snapshotted at 4.7.1. So, I made this version to be in sync with current telemetry version.